src/language-markdown/printer-markdown.js

```diff
@@ -503,9 +503,7 @@ function getNthListSiblingIndex(node, parentNode) {
   );
 }
 
-function getNthSiblingIndex(node, parentNode, condition) {
-  condition = condition || (() => true);
-
+function getNthSiblingIndex(node, parentNode, condition = (_) => true) {
```

Comment:
This default value not needed. Only used in this file, and it passed the argument.